drivers/timer/rk3568/timer.c

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2025, UNSW
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <stdint.h>
+#include <os/sddf.h>
+#include <sddf/timer/protocol.h>
+#include <sddf/util/util.h>
+#include <sddf/util/printf.h>
+#include <sddf/resources/device.h>
+
+__attribute__((__section__(".device_resources"))) device_resources_t device_resources;
+
+#define CLIENT_CH_START 2
```

Comment:
This shouldn't be necessary in current versions of the protocol - the config struct (see other examples for reference) tells us which channels are IRQs / the virt and we can assume all others are clients

Reply:
Done